clang/lib/Sema/SemaInit.cpp

```diff
@@ -6496,6 +6496,17 @@ static bool canPerformArrayCopy(const InitializedEntity &Entity) {
   return false;
 }
 
+static const FieldDecl *getConstField(const RecordDecl *RD) {
+  for (const FieldDecl *FD : RD->fields()) {
+    QualType QT = FD->getType();
+    if (QT.isConstQualified())
+      return FD;
+    if (const auto *RD = QT->getAsRecordDecl())
+      return getConstField(RD);
```

Comment:
Should we check the return value and if it is `nullptr` continue the outer for?

Reply:
Good call, I think we should. I've updated.